taxonomies/ingredients_processing.txt

```diff
@@ -1772,6 +1794,16 @@ fr:irradié, irradiée, irradiés, irradiées
 en:non-irradiated, not treated with irradiation, without irradiation
 fr:non irradié, non irradiée, non irradiés, non irradiées, sans irradiation
 
+#en:description:complete or partial removal of impurities in sugars, oils, fats and other natural materials by chemical/physical treatment
+en:refined, partially refined
+fr:raffiné, partiellement raffiné
```

Comment:
For matching, we need all gender / plural variants (e.g. to match "huile raffinée")

```suggestion
fr:raffiné, partiellement raffiné,raffinée, partiellement raffinée, raffinés, partiellement raffinés, raffinées, partiellement raffinées
```